data/transactions/logic/TEAL_opcodes.md

```diff
@@ -1104,6 +1122,15 @@ Fails unless the last instruction executed was a `callsub`.
 - branch to the Ath label. Continue at following instruction if index A exceeds the number of labels.
 - Availability: v8
 
+## match target ...
+
+- Opcode: 0x8e {uint8 branch count} [{int16 branch offset, big-endian}, ...]
+- Stack: ..., [N items] &rarr; ...
+- construct a list of match cases from B[0] to B[N] and branch to the Ith label where B[I] = A. Continue to the following instruction if no matches are found.
```

Comment:
`B[0]` to `B[N]` implies N+1 targets.

I'd avoid talk of "construct a list"

The input stack doesn't seem right. It should have N items and a value, right?

Reply:
You're right, seems I've been sloppy with the docs.